src/plugins/tools/lineReader/compoundMask.js

```diff
@@ -750,6 +750,7 @@ export default function compoundMaskFactory(options, dimensions, position) {
                     .setSize(size.width - options.resizeHandleSize * 2, size.height + options.resizeHandleSize * 2);
             },
 
+            // eslint-disable-next-line no-unused-vars
```

Comment:
I had to insert a few of these ignores for unused function params. I think it can be nice to leave params to document what the function does/did; if someone disagrees I can remove them though.